packages/test-utils/src/index.ts

```diff
@@ -160,8 +187,13 @@ function teardownMongoMemoryServer() {
   return stopping;
 }
 
-function _keystoneRunner(adapterName, tearDownFunction) {
-  return function (setupKeystoneFn, testFn) {
+type Setup = { keystone: Keystone<string> | BaseKeystone; context: KeystoneContext };
```

Comment:
`KeystoneContext` is the wrong type, no? It's for the new interfaces context which is quite different to current keystone context?

Reply:
No, this is for the new interfaces. The test-utils package support both old and new packages.

Reply:
So you're only planning to make tests for the new interfaces use TS?

Reply:
For now, yes.